webui/react/src/types.ts

```diff
@@ -253,11 +253,8 @@ export type HyperparametersFlattened = {
 };
 
 export enum ExperimentSearcherName {
-  AdaptiveAdvanced = 'adaptive',
```

Comment:
Do you use this to parse experiment configs that you get from the master?  If so, this would be high-risk, because old experiments might fail to load if they were created when PBT was allowed.